src/main/java/com/arckenver/nations/listener/BuildPermListener.java

```diff
@@ -16,17 +16,15 @@
 import com.arckenver.nations.ConfigHandler;
 import com.arckenver.nations.DataHandler;
 import com.arckenver.nations.LanguageHandler;
+import org.spongepowered.api.world.World;
 
 public class BuildPermListener
 {
 
 	@Listener(order=Order.FIRST)
 	public void onPlayerPlacesBlock(ChangeBlockEvent.Place event, @First Player player)
 	{
-		if (!ConfigHandler.getNode("worlds").getNode(event.getTargetWorld().getName()).getNode("enabled").getBoolean())
-		{
-			return;
-		}
+
```

Comment:
Why did you remove this part where we check if the plugin is enabled in the world?

If you think doing this elsewhere is better we can discuss it, but in that case, we would need to change it everywhere for consistency sake